src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml

```diff
@@ -75,3 +75,12 @@
     ansible_ssh_pipelining: true
   shell: "{{postgres_install_dir}}/bin/psql -U postgres -p {{postgresql_port}} {{postgresql_dbs.form_processing.proxy.name}} -c \"GRANT USAGE on FOREIGN DATA WRAPPER plproxy to \\\"{{postgresql_dbs.form_processing.proxy.user}}\\\"\""
   when: postgresql_dbs.form_processing and postgresql_dbs.form_processing.proxy.host == inventory_hostname
+
+- name: Set vm.dirty_background_bytes
+  become: yes
+  sysctl:
+    name: vm.dirty_background_bytes
+    value: '{{ dirty_background_bytes }}'
+    state: present
+    sysctl_file: /etc/sysctl.d/90-commcare-cloud.conf
```

Comment:
We don't really have a custom sysctl_file. I decided to just have this here for now, but it could potentially be a variable later. Not sure if we would want it to be all of commcare-cloud or split by service